chore(curriculum): add user centered design transcripts (#58332)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/25-front-end-development/lecture-user-centered-design/672bb03999f39379f67d8972.md

Watch the lecture video and answer the questions below.

# --transcript--

> What Is Progressive Disclosure?

Reviewer: ```suggestion
What is progressive disclosure?
```

> A progressive disclosure is a design pattern used to only show users relevant content based on their current activity and hide the rest. This is done to reduce cognitive load and make the user experience more intuitive.
> 
> For example, when you use Google's search page, most of your searches will be simple. So having a simple user interface with very few options makes sense.
> 
> But for the few times you will need a more advanced search, Google provides an advanced search option that will reveal more options.
> 
> You can find Google's advanced search option from the settings menu on the Google homepage. Or you can go to google dot com forward slash advanced underscore search.
> 
> Another example of progressive disclosure would be the "More details" button on products in an e-commerce website.
> 
> When you visit a site like Amazon, you will see a list of product images in different categories. This allows users to shop with ease and not get overwhelmed by too much information.
> 
> If the user is interested in a particular product, they can click on the product image to reveal more details about the product.
> 
> Displaying all of the information right away may be overwhelming for the user. So, by using progressive disclosure, the user can choose to see more information when they are ready.
> 
> So, what are some best practices for designing with the progressive disclosure technique?
> 
> The first consideration is to keep all important information visible at all times. Users should not have to struggle to find key information because it is hidden by extra advanced features.
> 
> Another consideration is to provide a single access point for users to access additional features or information. This could be a button or link that is always visible on the page.
> 
> Like in the earlier Amazon example, the additional product information was only made available when the user clicked on the product they were interested in.
> 
> There weren't multiple buttons or links to click on to see the additional information. Adding multiple access points can be confusing for users and can lead to a poor user experience. 
> 
> As you design your applications, think about the overall user experience and how you can use progressive disclosure to make it easier for users to find the information they need. 
> 
> When done correctly it can make a big difference to the overall user experience.

Reviewer: ```suggestion
A progressive disclosure is a design pattern used to only show users relevant content based on their current activity and hide the rest. This is done to reduce cognitive load and make the user experience more intuitive.

For example, when you use Google's search page, most of your searches will be simple. So having a simple user interface with very few options makes sense. But for the few times you will need a more advanced search, Google provides an advanced search option that will reveal more options.

You can find Google's advanced search option from the settings menu on the Google homepage. Or you can go to `google.com/advanced_search`.

Another example of progressive disclosure would be the "More details" button on products in an e-commerce website. When you visit a site like Amazon, you will see a list of product images in different categories. This allows users to shop with ease and not get overwhelmed by too much information.

If the user is interested in a particular product, they can click on the product image to reveal more details about the product. Displaying all of the information right away may be overwhelming for the user. So, by using progressive disclosure, the user can choose to see more information when they are ready.

So, what are some best practices for designing with the progressive disclosure technique? The first consideration is to keep all important information visible at all times. Users should not have to struggle to find key information because it is hidden by extra advanced features.

Another consideration is to provide a single access point for users to access additional features or information. This could be a button or link that is always visible on the page. Consider again Amazon's site: the additional product information is only made available when the user click on the product they are interested in. There aren't multiple buttons or links to click on to see the additional information. Adding multiple access points can be confusing for users and can lead to a poor user experience. 

As you design your applications, think about the overall user experience and how you can use progressive disclosure to make it easier for users to find the information they need. When done correctly it can make a big difference to the overall user experience.
```
Slightly modified the reference to the previous Amazon example.
